app/server/tests/redis.test.ts

```diff
@@ -15,30 +15,26 @@ describe("redis", () => {
     });
 
     afterEach(() => {
-        jest.clearAllMocks();
+        vi.clearAllMocks();
     });
 
-    it("can connect", (done) => {
-        const errorCallback = jest.fn();
+    it("can connect", async () => {
+        const errorCallback = vi.fn();
         const redis = redisConnection("redis://localhost:6379", errorCallback);
-        setTimeout(() => {
-            redis.disconnect();
-            expect(mockConsoleLog).toBeCalledTimes(1);
-            expect(mockConsoleLog.mock.calls[0][0])
-                .toBe("Connected to Redis server redis://localhost:6379");
-            expect(errorCallback).not.toHaveBeenCalled();
-            done();
-        }, 200);
+        await new Promise(res => setTimeout(res, 200));
```

Comment:
instead of setTimeout and using `done` callback (which is now completely deprecated) we just await 200ms before moving on